src/SonarQube.Client/Models/Organizations.cs

```diff
@@ -18,18 +18,40 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System.Diagnostics;
+
 using Newtonsoft.Json;
 
-namespace SonarLint.VisualStudio.Integration.Service
+namespace SonarQube.Client.Models
 {
-    [DebuggerDisplay("Key: {Key}, Name: {Name}")]
-    internal class OrganizationInformation
+    public class Organization
+    {
+        public string Key { get; }
+        public string Name { get; }
+
+        public Organization(string key, string name)
+        {
+            Key = key;
+            Name = name;
+        }
+
+        public static Organization FromDto(OrganizationDTO dto)
+        {
+            return new Organization(dto.Key, dto.Name);
+        }
+    }
+
+    public class OrganizationDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class 'OrganizationDTO' to match camel case naming rules, consider using 'OrganizationDto'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS101)